cmd/cluster/create.go

```diff
@@ -135,6 +134,16 @@ func NewCreateCommand() *cobra.Command {
 }
 
 func CreateCluster(ctx context.Context, opts Options) error {
+	annotations := map[string]string{}
+	for _, s := range opts.Annotations {
+		pair := strings.Split(s, "=")
```

Comment:
i'd use SplitN to allow the value of the annotation to contain `=`'s

Reply:
Good catch — fixed